scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -14,6 +14,7 @@
     become: yes
     shell: |
       /usr/sbin/growpart /dev/vda 1 && /sbin/resize2fs /dev/vda1
+    ignore_errors: True
```

Comment:
What kind of error do you want to ignore? Is the package not installed?

Reply:
when reusing an existing stack, growpart reports there is nothing to grow, which is taken as error.
Sorry I should have mentioned that in the description